docs/CHANGELOG.md

```diff
@@ -10,6 +10,10 @@ nav_order: 5
 
 ## main
 
+* Include ViewComponent::UseHelpers by default instead.
```

Comment:
Instead of what?

Reply:
thank you @Spone, @joelhawksley has fixed this